test/test_sparse_csr.py

```diff
@@ -2004,33 +2032,102 @@ def test_compressed_layout_conversions_coverage(self, device, from_layout, to_la
         that an exception is thrown for unsupported conversions.
         """
 
-        def _convert_to_layout(a, target_layout):
-            if target_layout is torch.sparse_csr:
-                return a.to_sparse_csr()
-            if target_layout is torch.sparse_csc:
-                return a.to_sparse_csc()
-            if target_layout is torch.sparse_bsr:
-                return a.to_sparse_bsr((2, 2))
-            if target_layout is torch.sparse_bsc:
-                return a.to_sparse_bsc((2, 2))
-            raise NotImplementedError(repr(a))
-
         def _to_from_layout(layout_a, layout_b):
             a = make_tensor((6, 10), dtype=torch.float, device=device)
             expect_error = (layout_a in [torch.sparse_csc, torch.sparse_bsc]
                             or layout_b in [torch.sparse_csc, torch.sparse_bsc])
             expect_error = expect_error or (layout_a, layout_b) == (torch.sparse_bsr, torch.sparse_bsr)
             expect_error = expect_error or (layout_a, layout_b) == (torch.sparse_bsr, torch.sparse_csr)
+            # CSC to CSR conversion is supported
+            if layout_a is torch.sparse_csc and layout_b is torch.sparse_csr:
+                expect_error = False
             if expect_error:
                 with self.assertRaises(RuntimeError):
-                    b = _convert_to_layout(a, layout_a)
-                    _convert_to_layout(b, layout_b)
+                    b = self._convert_to_layout(a, layout_a)
+                    self._convert_to_layout(b, layout_b)
             else:
-                b = _convert_to_layout(a, layout_a)
-                _convert_to_layout(b, layout_b)
+                b = self._convert_to_layout(a, layout_a)
+                self._convert_to_layout(b, layout_b)
```

Comment:
For sanity, I think we should also compare the actual conversion results. For instance, via
```suggestion
                c = self._convert_to_layout(b, layout_b)
                self.assertEqual(a.to_dense(), c.to_dense())
```

Reply:
Not all layouts support conversion to Strided yet, but I'll enable it for those that do.